qiskit/tools/visualization/_matplotlib.py

```diff
@@ -156,7 +197,10 @@ def _gate(self, xy, fc=None, wide=False, text=None, subtext=None):
         if fc:
             _fc = fc
         elif text:
-            _fc = self._style.dispcol[text]
+            try:
```

Comment:
It might be neater to do `if text in self._style.dispcol` here rather than catching the error

Reply:
Done